go.mod

```
@@ -35,6 +36,7 @@ require (
 	github.com/weaveworks/common v0.0.0-20190410110702-87611edc252e
 	github.com/weaveworks/go-checkpoint v0.0.0-20170503165305-ebbb8b0518ab
 	github.com/whilp/git-urls v0.0.0-20160530060445-31bac0d230fa
+	go.mozilla.org/sops v0.0.0-20190912205235-14a22d7a7060
```

I think it would be clearer if we depended on an explicit version of `sops`

```suggestion
	go.mozilla.org/sops v3.4.0
```

(`14a22d7a7060` and `v3.4.0` point to the same commit)

---

Yes, but that fails with, due to the semantic versioning requirement of go modules. This will be fixed in the future when https://github.com/mozilla/sops/pull/566 is included in a release.